README.md

```diff
@@ -1 +1,95 @@
-# budget-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Most of your tests are failing, kindly ensure all unit and integration tests are passing.

Reply:
Done